test/transformationsTest.js

```diff
@@ -14,6 +14,26 @@ test.test('Transformations', t => {
       t.equal(result, expected)
       t.end()
     })
+
+    t.test('should return unchanged val if not in val is in unexpected format', t => {
```

Comment:
I think the title of this test might is a bit confusing. `... if not in val is in ...`

Reply:
Fixed :100: